legion/legion/io.py

```diff
@@ -464,3 +453,65 @@ def prepare_func(input_dict):
     send_header_to_stderr(legion.containers.headers.MODEL_VERSION, version)
 
     return model
+
+
+def export_df(apply_func, input_data_frame,
+              filename=None, prepare_func=None,
```

Comment:
There should be a way to override type of input_data_frame columns. The sigunature should be:
export_df(apply_func, 
                 column_types=None,
                 filename=None,
                 ...)
So, invocation will look:
              export_df(apply, column_types=DFColumns(input_data_frame))